src/lib.rs

```diff
@@ -157,60 +155,59 @@
 //!
 //! ### Non-local flavor
 //!
-//! The non-local flavors can be used between arbitrary tasks and threads.
-//! They use internal synchronization for this in form of an embedded `Mutex` in
-//! the form of a [`parking_lot::Mutex`] type.
+//! The non-local flavors can be used between arbitrary tasks and threads.  They
+//! use internal synchronization for this in form of an embedded `Mutex` of
+//! [`parking_lot::Mutex`] type.
 //!
 //! The non-local flavors are only available in `std` environments.
 //!
 //! ### Shared flavor
 //!
-//! For some types also a shared flavor of the type is provided. Non-local
-//! flavors of types are `Sync`, but still can only be shared by reference
-//! between various tasks.
-//! Shared flavors are also `Sync`, but the types do also implement the `Clone`
-//! trait, which allows to duplicate the object, and pass the ownership of it
-//! to a different task. These types allow to avoid references (and thereby
-//! lifetimes) in some scenarios, which makes them more convenient to use.
-//! The types also return `Future`s which do not have an associated lifetime.
-//! This allows to use those types as implementations of traits, without the
-//! need for generic associated types (GATs).
-//!
-//! Due to the requirement of atomic reference counting, those types are
+//! For some types a shared flavor is provided. Non-local flavors of types are
+//! `Sync`, but they still can only be shared by reference between various tasks.
+
+//! Shared flavors are also `Sync`, but the types additionally implement the
+//! `Clone` trait, which allows them to duplicate the object, and to pass
```

Comment:
`them`? It's not the types that duplicate the object, but some other code, right?

Reply:
Good catch.  I've fixed this and force-pushed the branch.